src/bot/exts/dragonfly/dragonfly.py

```diff
@@ -131,7 +131,7 @@ async def report(self: Self, interaction: discord.Interaction, button: discord.u
 def _build_package_scan_result_embed(scan_result: PackageScanResult) -> discord.Embed:
```

Comment:
This function name should be changed to something that actually specifies that it's a confirmed malicious result

Reply:
`_build_malicious_package_scan_result_embed`? Feels a bit lengthy though

Reply:
Hmm. I'll just leave this alone for now, and bring it up for discussion later.